relay-general/src/store/normalize.rs

```diff
@@ -877,6 +877,15 @@ pub fn light_normalize_event(
             );
         }
 
+        if config.enrich_spans {
+            extract_span_tags(
+                event,
+                &tag_extraction::Config {
+                    max_tag_value_size: config.max_tag_value_length,
+                },
+            );
+        }
```

Comment:
Had to move this from `TransactionsProcessor` because device class normalization runs after `TransactionsProcessor`.